cpp/src/gandiva/gdv_function_stubs.cc

```diff
@@ -1000,5 +1172,47 @@ void ExportedStubFunctions::AddMappings(Engine* engine) const {
   engine->AddGlobalMappingForFunc("gdv_fn_sha256_decimal128",
                                   types->i8_ptr_type() /*return_type*/, args,
                                   reinterpret_cast<void*>(gdv_fn_sha256_decimal128));
+
+  // gdv_fn_utf8_char_length
+  args = {
```

Comment:
you don't need to register these here. only upper/lower is enough.

Reply:
Functions removed from global mapping.